src/index.js

```diff
@@ -0,0 +1,142 @@
+let currentTemperature = 0;
+
+const tempValue = document.getElementById('tempValue');
+
+const temperatureDisplay = () => {
+    tempValue.textContent = currentTemperature;
+    tempColor();
+};
+
+const addDegree = () => {
+    currentTemperature += 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const subtractDegree = () => {
+    currentTemperature -= 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const tempColor = () => {
+    const BackgroundColor = document.getElementById('currentTempButton');
+
+    const elementFontColor = document.getElementById('tempValue');
+    let currentColor;
+    
+    const elementLandscape = document.getElementById('landscape');
+    let currentLandscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+
+    if (currentTemperature <= 49) {
+        currentColor = '#30849D';
+    } else if (currentTemperature >= 50 && currentTemperature <= 59 ) {
+        currentColor = '#50A63F';
+    } else if (currentTemperature >= 60 && currentTemperature <= 69 ) {
+        currentColor = '#FEE086';
+        currentLandscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (currentTemperature >= 70 && currentTemperature <= 79 ) {
+        currentColor = '#F8B5A2';
+        currentLandscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (currentTemperature >= 80) {
+        currentColor = '#E06767';
+        currentLandscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    }
+    BackgroundColor.style.backgroundColor = currentColor;
+    elementFontColor.style.color = currentColor;
+    elementLandscape.textContent = currentLandscape;
+};
+
+const citySky = (event) => {
+    const selectedOption = event.target.value;
+    const elementSky = document.getElementById('sky');
+    let currentCitySky = '';
+
+    if (selectedOption === 'sunny') {
+        currentCitySky = '☀️ ☁️ ☀️ ☀️ ☁️ ☀️ ☁️ ☀️ ☀️ ☁️ ☀️';
+    } else if (selectedOption === 'cloudy') {
+        currentCitySky = '☁️☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️☁️';
+    } else if (selectedOption === 'rainy') {
+        currentCitySky = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    } else if (selectedOption === 'snowy') {
+        currentCitySky = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    }
```

Comment:
Same comment as above, consider using a data structure to hold your sky values and then iterate over the data structure to get the emojis so you don't need a bunch of conditional logic. When we have longer conditional logic and it could be refactored away, we should consider doing so because lots if /else ifs/else can be brittle.

```js
const skyOptions = [
    { name: 'sunny', display: '☀️ ☁️ ☀️ ☀️ ☁️ ☀️ ☁️ ☀️ ☀️ ☁️ ☀️' },
    { name: 'cloudy', display: '☁️☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️☁️'},
    { name: 'rainy', display: '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧' },
    { name: 'snowy', display: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨' }
];